content/academy/glossary/rate_limiting.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Rate-Limiting
+description: Learn about rate-limiting, a common tactic used by websites to avoid a large rate of requests from a single IP address.
+menuWeight: 6.2
+paths:
+    - glossary/rate-limiting
+---
+
+# Rate-Limiting
+
+In the past, most websites had their own anti-scraping solutions, the most common of which was **IP address rat-limiting**. In recent years, the popularity of third-party specialized anti-scraping providers has dramatically increased, but a lot of websites still use rate-limiting to only allow a certain number of requests per second/minute/hour to be sent from a single IP; therefore, crawler requests have the potential of being blocked entirely quite quickly. Thus, in cases when a higher number of requests is expected for the crawler, using a proxy and rotating the IPs is essential to let the crawler run as smoothly as possible and avoid being blocked.
```

Comment:
typo rate-limiting

Reply:
we should raise awareness for rat-limiting as well, though :)